content/articles/2018-08-tidyverse-developer-day.Rmarkdown

```diff
@@ -0,0 +1,41 @@
+---
+title: 'Save the date: tidyverse developer day'
+author: Mara Averick
+date: '2018-08-27'
+slug: tidyverse-developer-day
+description: > 
+  Save the date for tidyverse developer day the Saturday following rstudio::conf!
+categories:
+  - other
+photo:
+  url: https://unsplash.com/photos/ZMMXSRMSoI8
+  author: rawpixel
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+### Save the date `r emo::ji("calendar")`
+
+On the Saturday following [rstudio::conf](https://www.rstudio.com/conference/) 
+(19 January), we'll be holding our first ever **tidyverse developer day**, and you're all invited! So, if you're interested, plan your travel accordingly.
```

Comment:
If we have any concerns that interest will exceed capacity, we should find a different wording. Or decide the risk is one we're willing to take.

Reply:
I'll change the wording here, but I agree about the overall lack of detail that'll beg questions — I don't know the answers, but I'm happy to add any other info we have!